src/test/java/teammates/storage/api/FeedbackResponseMonitorDbTest.java

```diff
@@ -0,0 +1,58 @@
+package teammates.storage.api;
+
+import java.util.List;
+
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.attributes.FeedbackResponseRecordAttributes;
+import teammates.common.exception.EntityAlreadyExistsException;
+import teammates.common.exception.InvalidParametersException;
+import teammates.test.BaseComponentTestCase;
+
+/**
+ * SUT: {@link FeedbackResponseMonitorDb}.
+ */
+public class FeedbackResponseMonitorDbTest extends BaseComponentTestCase {
+    FeedbackResponseMonitorDb db = new FeedbackResponseMonitorDb();
+    long[] countTestData = new long[] { 105, 105, 105, 105, 105 };
+
+    private void populateTestData() throws EntityAlreadyExistsException, InvalidParametersException {
+        long currentTime = System.currentTimeMillis();
+        for (int i = 0; i < countTestData.length; i++) {
+            long timestamp = currentTime - 120 * i;
+            FeedbackResponseRecordAttributes attributes = new FeedbackResponseRecordAttributes(countTestData[i], timestamp);
+            db.createEntity(attributes);
```

Comment:
Can consider inlining the creation?
`db.createEntity(new FeedbackResponseRecordAttributes(countTestData[i], timestamp);`

Reply:
Down here I was saying inlining the object creation with the method invocation; the time stamp can prolly read better if you separate it.